### PR TITLE
Show package link to Github and number of stars in nav bar

### PR DESCRIPTION
**Pull request by b-per**
POC related to #1724 but with a different approach. Feel free to just close it if we don't want to implement it.

The link to dbt-core's repo in GitHub is removed from the main page but it is replaced by links to individual packages, showing the number of stars for each package.

In that case, the key difference between the link in the nav bar and the link in the body is that the nav bar points to the main/master branch when the body points to the actual branch the user has selected.

### Main page
image

### Package page
image

### CODE REVIEW

File: source/_nav.erb [hunk 1]
```diff
@@ -8,17 +8,19 @@
...
 			data-show-count="true"
-			aria-label="Star dbt-labs/dbt on GitHub"
+			aria-label="Star the package on GitHub"
 			target="_blank">
 			     <i data-icon="star"></i>
-			     GitHub
+			     <%= version._source.url.split("/")[3] %>/<%= version._source.url.split("/")[4] %>
 			</a>
 		</div>
+		<% end %>
```
Comment (dbeatty10) [inline source/_nav.erb:23]: This approach looks good to me.

The key thing we'd need to do is handle the case where `version._source.url` doesn't conform to the expected pattern. e.g., handle `nil` values or strings that don't split on `/` like we'd expect.

**Review by dbeatty10 — CHANGES_REQUESTED** [review]

**Comment by b-per** [comment]
I have added checks on the URL before showing the `<div>`

**Comment by dbeatty10** [comment]
@b-per did you push your latest commit(s)?

I'm only seeing a single commit:
image

**Comment by joellabes** [comment]
@b-per i love this! 

I wonder whether it should instead go 
Screen Shot 2022-08-26 at 1 31 03 PM

The get started button is "get started with packages in general", and I suspect that the name of the repo right beside it would imply that it's "get started with dbt_expectations"

**Comment by b-per** [comment]
@dbeatty10: My bad, I tried to push it yesterday but it got stuck and I didn't realize. I pushed it now

@joellabes: It makes sense! I have moved it across to the header of the package

image

File: source/package.template.html.erb [hunk 2]
```diff
@@ -32,6 +32,19 @@
                   <%= package.namespace %>
                 </a>
               </em><br/>
+              <% if is_valid_github_url(version._source.url) %>
```
Comment (dbeatty10) [inline source/package.template.html.erb:35]: This check is awesome.

I am still a little bit worried about a case where the full path of `version._source.url` isn't defined. But further down the code, it's already accessing that same reference without any checks, so we'll let it fly:
```
(<a href="<%= version._source.url %>">View on GitHub</a>)
```

**Review by dbeatty10 — APPROVED** [review]
Awarding 436 GitHub Stars ⭐ for you @b-per 🤩

**Pull request merged**